integration/admin-client-jee/src/main/java/org/keycloak/admin/client/resource/OrganizationsResource.java

```diff
@@ -39,9 +39,32 @@ public interface OrganizationsResource {
     @Path("{id}")
     OrganizationResource get(@PathParam("id") String id);
 
+    /**
+     * List all organizations in the realm.
+     *
+     * @return a list containing the organizations.
+     */
     @GET
     @Produces(MediaType.APPLICATION_JSON)
-    List<OrganizationRepresentation> getAll(
-            @QueryParam("domain-name") String domainName
+    List<OrganizationRepresentation> list();
```

Comment:
Isn't better to keep the `getAll` name?

Reply:
I tried to keep it consistent with what we have in the UserEnpoint, for example. But I don't really care much about the name of the methods in the resource interface, so I'll revert to `getAll`.